### ISSUE TITLE
Chapter 4 example not usable anymore

### ISSUE THREAD

**Issue by nitmws** [PacktPublishing/-Vue.js-3-By-Example#10]
I'm interested in the book Vue.js 3 By Examples but wanted to check the usability of the code examples first. So I cloned this repository and focused on the Chapter04 example: 

- ran npm install using node.js 18.12.1: saw many DEPRECATED and finally "70 vulnerabilities (2 low, 19 moderate, 34 high, 15 critical)"
- ran npm audit fix: "42 vulnerabilities (1 low, 7 moderate, 25 high, 9 critical)" and got EBADENGINE on  package: '@achrinza/node-ipc@9.2.2 only a node.js version 8 thru 17 is supported.
- updated all referenced node_modules in package.json to the latest version: "15 vulnerabilities (4 moderate, 11 high)" but still EBADENGINE and when running the serve package.json-script many problems with eslint, was not able to get it working.

Final conclusion: the latest commit on all these chapters was in April 2021, no update since then - now not usable anymore. So I cancelled my plans to buy Vue.js 3 By Examples.

**Comment by marabesi** [comment]
@nitmws have you tried to run the code sample without updating the dependencies?

**Comment by jauyeunggithub** [comment]
Yes. They run without updating the dependencies.

**Comment by nitmws** [comment]
@marabesi no, I have not tried that. But when I install packages for a node.js app it is not a option to run the app with multiple serious vulnerabilities. If a package is classified as vulnerable npmjs has good reasons for that.

**Comment by marabesi** [comment]
@nitmws keep in mind that those are examples for **learning** not a production application :)

**Comment by nitmws** [comment]
@marabesi I learn something for using it now, not 2 years ago. If a package cannot be used anymore it doesn't help to learn about an app using it.

**Comment by marabesi** [comment]
for further readers of this book, as of today, you can run the examples and get it working (as pointed by @jauyeunggithub )  by installing the dependencies with:

```
npm i
````

and then running it with:

````
npm run server

````

The output will be:

![image](https://user-images.githubusercontent.com/2129872/226918051-602645a4-687c-4075-8531-39b86363e6b5.png)

Keep in mind that those are learning examples, taking an app to production will require more than what the objective of the book is.

The example above was based on chapter 3, other chapters might require double-checking as well.
